openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1995,6 +1995,15 @@ The following additional Credential Issuer metadata parameters are defined for t
           * `locale`: OPTIONAL. String value that identifies language of this object represented as language tag values defined in BCP47 [@!RFC5646]. There MUST be only one object for each language identifier.
 * `order`: OPTIONAL. Array of the claim name values that lists them in the order they should be displayed by the Wallet.
 
+> The above metadata structure has some known limitations:
+>
+> * It cannot be used to describe claims in credentials that have the name `mandatory`, `value_type` or `display`.
+> * It is not possible to provide `mandatory`, `value_type` or `display` values for objects that contain claims 
```

Comment:
this is the most important and should be the first one. and it should also be much more clearer.
```suggestion
> * It is not possible to provide `mandatory`, `value_type` or `display` values for the intermediary claims in the nested objects
```